rotkehlchen/inquirer.py

```diff
@@ -467,15 +481,50 @@ def find_price(
         if ignore_cache is False:
             cache = instance.get_cached_current_price_entry(cache_key=(from_asset, to_asset))
             if cache is not None:
-                return cache.price
+                return cache.price, cache.oracle
 
-        oracle_price = instance._query_oracle_instances(
+        oracle_price, oracle_queried = instance._query_oracle_instances(
+            from_asset=from_asset,
+            to_asset=to_asset,
+            skip_onchain=skip_onchain,
+            coming_from_latest_price=coming_from_latest_price,
+        )
+        return oracle_price, oracle_queried
+
+    @staticmethod
+    def find_price(
+            from_asset: Asset,
+            to_asset: Asset,
+            ignore_cache: bool = False,
+            skip_onchain: bool = False,
+            coming_from_latest_price: bool = False,
+    ) -> Price:
+        """Wrapper around _find_price to ignore oracle queried when getting price"""
+        price, _ = Inquirer()._find_price(
             from_asset=from_asset,
             to_asset=to_asset,
+            ignore_cache=ignore_cache,
+            skip_onchain=skip_onchain,
+            coming_from_latest_price=coming_from_latest_price,
+        )
+        return price
+
+    @staticmethod
+    def find_price_and_oracles(
```

Comment:
this plural here should also have been addressed... but at this point I am just tired of repeating myself in reviews and don't care. Dissapointed ...